clusterloader2/testing/load/modules/scheduler-throughput.yaml

```diff
@@ -68,33 +55,15 @@ steps:
         Group: scheduler-throughput
         CpuRequest: 1m
         MemoryRequest: 10M
-{{if $ENABLE_HUGE_SERVICES}}
-        SvcName: huge-service
+{{if $serviceName}}
```

Comment:
Nit: You can avoid if by "SvcName: {{$serviceName}}".
It will work just fine for empty value

Reply:
Done.